src/shared/icons.utils.js

```diff
@@ -20,8 +20,13 @@
  */
 
 const { nativeTheme } = require('electron')
+const { isLinux } = require('./os.utils.js')
 
 const icons = {
+	window: {
+		linux: require('../../img/icons/icon.png'),
+	},
```

Comment:
I'd add here a comment here: `Executable's icon is used by default`

Reply:
Done